src/server/routes/api.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express')
+const request = require('request')
+const router = express.Router()
+const Transaction = require('../models/transactionSchema')
+
+
+
+router.get('/transactions', async function (req, res) {
+    let transactions = await Transaction.find({})
+    res.send(transactions)
+})
+
+router.post('/transaction', async function(req, res) {
+    let transaction = new Transaction(req.body)
+    console.log(transaction)
+    await transaction.save()
+    res.end()
+})
+
+router.delete('/transaction/:id', async function(req, res) {
+    let transactionId = req.params.id
```

Comment:
Use _const_ instead of _let_ (when you are not planning to allow value change), It's especially relevant with ids...